Author: Olivier Sallou <osallou@debian.org>
Subject: ignore tests needing a running consul instance
Description: some tests require a running instance of Consul,
 so disabling them for Debian CI
Last-Updated: 2017-08-17
Forwarded: no
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -20,7 +20,7 @@
     p = os.path.join(os.path.dirname(__file__), 'test_aio.py')
     collect_ignore.append(p)
 
-
+'''
 if sys.version_info[0] == 2 and sys.version_info[1] < 7:
     p = os.path.join(os.path.dirname(__file__), 'test_twisted.py')
     collect_ignore.append(p)
@@ -28,6 +28,7 @@
     collect_ignore.append(p)
 else:
     pytest_plugins = "pytest_twisted"
+'''
 
 
 def get_free_ports(num, host=None):
